### PR TITLE
Replace deprecated ugettext with gettext_lazy

### PR DESCRIPTION
**Pull request by vleL**

### CODE REVIEW

File: pyas2/urls.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 from django.urls import path
 from django.contrib.auth.decorators import login_required
+from django.urls import include, path
```
Comment (abhishek-ram) [inline pyas2/urls.py:3]: why these changes?

Comment (vleL) [reply pyas2/urls.py]: That was committed by mistake. I removed it.

**Comment by robincsamuel** [comment]
@abhishek-ram could you please release the pip package with this update? :)

Thanks!

**Comment by abhishek-ram** [comment]
v1.2.2 has been released
